control/controlclient/map.go

```diff
@@ -85,6 +91,7 @@ func (ms *mapSession) addUserProfile(userID tailcfg.UserID) {
 // information from prior MapResponse values.
 func (ms *mapSession) netmapForResponse(resp *tailcfg.MapResponse) *netmap.NetworkMap {
 	undeltaPeers(resp, ms.previousPeers)
+	ms.flagExpiredPeers(resp)
```

Comment:
we probably need to call this with a timer for cases where control doesn't send netmap updates for a long time (or is unreachable).

Reply:
Yep, planning on following up with that in another PR; it's a bit tricky since I think we need to create a new `NetMap`, since we can't mutate the existing one